openstack/placement/v1/resourceproviders/results.go

```diff
@@ -33,6 +33,37 @@ type ResourceProvider struct {
 	RootProviderUUID string `json:"root_provider_uuid"`
 }
 
+type Usage struct {
+	DiskGB   int `json:"DISK_GB"`
+	MemoryMB int `json:"MEMORY_MB"`
+	VCPU     int `json:"VCPU"`
+}
+
+type ResourceProviderUsage struct {
+	ResourceProviderGeneration int   `json:"resource_provider_generation"`
+	Usages                     Usage `json:"usages"`
+}
+
+type Inventory struct {
```

Comment:
Similar to my comment in #1861:

This appears to be well defined: https://github.com/openstack/placement/blob/55635d918ce32667352fb94e4ae3e388069dd9c1/placement/schemas/inventory.py

So maybe this would work better?

```go
type ResourceProviderInventories struct {
	ResourceProviderGeneration int                  `json:"resource_provider_generation"`
	Inventories                map[string]Inventory `json:"inventories"`
}
```

Reply:
I agree.  I'll make that change.